code/game/objects/items/weapons/dna_injector.dm

```diff
@@ -1190,3 +1190,35 @@
 	New()
 		block = REMOTETALKBLOCK
 		..()
+
+/obj/item/weapon/dnainjector/nofail/randompower/New(newloc)
+    var/type = pick(/obj/item/weapon/dnainjector/nofail/hulkmut,
```

Comment:
And this is exactly what I'd have to do.  Re-write the entire file https://github.com/Probe1/vgstation13/blob/76b0cf46fbed6b876a355de2ebccd5b972f93f56/code/game/objects/items/weapons/dna_injector.dm and divide everything into: Power injector, Power removal injector, Defect injector, defect removal injector

I mean do I really have to rewrite an _entire file just for one damn thing?_